pandas/tests/extension/base/dtype.py

```diff
@@ -51,10 +50,6 @@ def test_eq_with_numpy_object(self, dtype):
     def test_array_type(self, data, dtype):
         assert dtype.construct_array_type() is type(data)
 
-    def test_array_type_with_arg(self, data, dtype):
```

Comment:
This test could never pass or fail correctly. It would always raise a TypeError since the method doesn't take an argument.